content/css/app.less

```diff
@@ -274,49 +274,53 @@
 
 .metric-widget,
 .metric-detail {
-    display: inline-block;
+    display: inline-flex;
     padding: 0px;
 }
 
 .metric-widget > button {
-    padding: 5px 2px 5px 2px;
     border: 0px #000;
     width: 100%;
-    background-color: @color-deep-azure;
-    color: #fff;
-    vertical-align: top;
+    background-color: @color-deep-slate;
+    color: @color-lighter-slate;
 }
 
 .metric-widget {
-    width: 100px;
-    margin: 0 2px;
-
-    /*
-    These styles are optimized for 3 KPI widgets on the top.
-    The total margin for 3 widgets will be 12px.
-    The ideal width for each widget will be slightly less than: (minWidth - 12px) / 3
-    */
-    @media (min-width: @mobile-landscape) {
-        width: 155px;
-    }
-
-    @media (min-width: @tablet-portrait) {
-        width: 250px;
-    }
-
-    @media (min-width: @tablet-landscape) {
-        width: 335px;
-    }
+    width: 33.3%;
+}
 
-    @media (min-width: @desktop) {
-        width: 420px;
-    }
+/*
+These styles can determine the number of siblings the metric-widget has (up to three)
+and was adapted from http://stackoverflow.com/questions/8720931/can-css-detect-the-number-of-children-an-element-has
+*/
+// One item
+.metric-widget:first-child:nth-last-child(1) {
+  width: 100%;
```

Comment:
I tried working with flexbox and utilizing the flex sizing, but didn't like how disorganized the widgets would turn out so I opted to not go for it. The sizing of the widgets supplemented with adding in borders between the widgets made the section look sloppy. I created a topic branch with the modifications that I could make trying to get a decent flexbox use in (topic-kpiSoHoFlex). I don't think I would want to use flex grow in this case because I'm cascading width based upon how many elements are in the row.